core_lib/helpers/logging.py

```diff
@@ -1,20 +1,31 @@
 import logging
 from functools import wraps
 
-from core_lib.helpers.func_utils import build_value_by_func_parameters, get_calling_module
+from core_lib.helpers.func_utils import (
+    build_value_by_func_parameters,
+    generate_key_by_func_parameters,
+)
 
 
 class Logging(object):
-    def __init__(self, message: str = '', level: int = logging.INFO, stack_depth=1):
+    def __init__(self, message: str = '', level: int = logging.INFO, stack_depth=1, log_parameters: bool = False):
         self.message = message
         self.level = level
-        self.calling_module = get_calling_module(stack_depth)
+        self.log_parameters = log_parameters
 
     def __call__(self, func, *args, **kwargs):
         @wraps(func)
         def __wrapper(*args, **kwargs):
-            message = build_value_by_func_parameters(self.message, func, *args, **kwargs)
-            logging.getLogger(self.calling_module).log(self.level, '{}.{}'.format(self.message, ''.join(message)))
+            formatted_params = ""
+            if self.log_parameters:
+                formatted_params = build_value_by_func_parameters(
+                    generate_key_by_func_parameters(func, *args, **kwargs), func, *args, **kwargs
+                )
+
+            logging.basicConfig(level=self.level)
+            logging.getLogger(func.__qualname__).log(

```

Comment:
1. instead of `__qualname__` here use `__name__`
2. `if not self.message` use the `generate_key_by_func_parameters(func, *args, **kwargs)`  (remove from `formatted_params`)